code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -187,7 +187,7 @@
 /obj/structure/destructible/clockwork/massive/ratvar
 	name = "Ratvar, the Clockwork Justiciar"
 	desc = "<span class='userdanger'>What is what is what are what real what is all a lie all a lie it's all a lie why how can what is</span>"
-	clockwork_desc = "<span class='large_brass'><b><i>Ratvar, the Clockwork Justiciar, your master eternal.</i></b></span>"
+	clockwork_desc = "It's Ratvar. You kind of expected more fanfare."
```

Comment:
I always thought that it was just your mind trying to comprehend the fact that a literal god is in front of you. I'll update them both.